### ISSUE TITLE
[med] JAWS+Chrome: About: Several links announced without link role

### ISSUE THREAD

**Issue by ogumen** [Expensify/App#5459]
## Action Performed:
Prerequisites: the user is logged in.

1. Using JAWS+Chrome, open https://staging.new.expensify.com/
2. Navigate to Settings > Profile > About.
3. Press Tab key to focus the links 'View the code' and 'View open jobs'.
4. Note how the elements are announced.

## Expected Result:
The links should be announced with the link role.

## Actual Result:
The elements open a new page but are announced without any role.

HTML:
`<div tabindex="0" class="css-1dbjc4n r-1loqt21 r-1otgn73 r-13qz1uu" style="border-radius: 0px; flex-direction: row; justify-content: space-between; padding: 12px 20px; -webkit-box-orient: horizontal; -webkit-box-direction: normal; -webkit-box-pack: justify;"><div class="css-1dbjc4n" style="flex-direction: row; -webkit-box-orient: horizontal; -webkit-box-direction: normal;"><div class="css-1dbjc4n r-1awozwy r-1777fci" style="height: 40px; width: 40px;">...<div dir="auto" class="css-901oao css-bfa6kz" style="color: rgb(11, 27, 52); font-family: GTAmericaExp-Regular; font-size: 15px; font-weight: 700; line-height: 20px; margin-left: 12px; max-width: 240px;">View the code</div></div></div><div class="css-1dbjc4n" style="flex-direction: row; min-height: 40px; -webkit-box-orient: horizontal; -webkit-box-direction: normal;">...</div>`

## Workaround:
No workaround

## Area issue was found in:
About overlay

## Failed WCAG checkpoints
4.1.2

## User impact:
The screen reader users will not be informed that the focused element is a link.

## Suggested resolution:
Implement the elements as native links using `<a>` tag with a non-empty href attribute, so that their role announces correctly. See: https://www.a11yproject.com/posts/2019-02-15-creating-valid-and-accessible-links/

## Platform:
Where is this issue occurring?

- Web

**Version Number:**
**Reproducible in staging?:**Yes
**Reproducible in production?:**
**Notes/Photos/Videos:** Any additional supporting documentation
**Issue reported by:**

https://user-images.githubusercontent.com/88733897/134561009-f0590387-d66b-460e-873f-dcd566ef9838.mp4

![Bug5180535_Several_links_announced_without_link_role](https://user-images.githubusercontent.com/88733897/134561019-2f889438-70fa-497a-9611-b6b0e638afc7.png)

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)
